docs/index.rst

```diff
@@ -76,27 +111,40 @@ Text Visualization
 - :doc:`api/text/umap_vis`: plot similar documents closer together to discover clusters
 - :doc:`api/text/postag`: plot the counts of different parts-of-speech throughout a tagged corpus
 
-... and more! Visualizers are being added all the time; be sure to check the examples (or even the `develop branch <https://github.com/DistrictDataLabs/yellowbrick/tree/develop>`_) and feel free to contribute your ideas for new Visualizers!
+... and more! Visualizers are being added all the time. Check the examples (or even the `develop branch <https://github.com/DistrictDataLabs/yellowbrick/tree/develop>`_). Feel free to contribute your ideas for new Visualizers!
 
 Getting Help
 ------------
 
-Yellowbrick is a welcoming, inclusive project in the tradition of matplotlib and scikit-learn. Similar to those projects, we follow the `Python Software Foundation Code of Conduct <http://www.python.org/psf/codeofconduct/>`_. Please don't hesitate to reach out to us for help or if you have any contributions or bugs to report!
+Can't get someting to work? Here are places to search for answers:
```

Comment:
I really like that you put this into a list.  In the spirit of streamlining (a favorite activity of ours!), perhaps we could combine them to streamline it further?  IE
1.)The docs...
2.)Stackoverflow.  Note: if you can't find a solution, ask a question and tag it (also Reddit is growing in popularity too)...
3.)YB Google groups
4.)Tweet at us...

Reply:
Good thought. I've combined the lists.